### PR TITLE
dx-1162 Card Payments PayeeReference string

### PR DESCRIPTION
**Pull request by arebra**
Tried to ad an if so that PayeeReference for card payments has string(50) and the rest have string(30).

### CODE REVIEW

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6110913&bid=20001000)

File: _includes/payee-reference.md [hunk 1]
```diff
@@ -17,10 +17,17 @@ specific processing rules depending on specifications in the contract.
...
+        {% if api_resource == "card" %}
     1.  If you select *Option A* in the settlement process (Swedbank Pay will
-        handle the settlement), Swedbank Pay will send the `transaction.number` to
-        the acquirer and the `payeeReference` must be in the format of characters
-        `A-Za-z0-9` (including `-`) and `string(30)`.
+        handle the settlement), Swedbank Pay will send the `transaction.number`
+        to the acquirer and the `payeeReference` must be in the format of
+        characters `A-Za-z0-9` (including `-`) and `string(50)`.
+        {% else %}
+    1.  If you select *Option A* in the settlement process (Swedbank Pay will
```
Comment (payex-dev) [inline _includes/payee-reference.md:26]: Codacy found an issue: [[ordered-list-marker-value] Marker should be `2`, was `1`](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6110913)

**Review by NullableInt — COMMENTED** [review]
Insted of duplicating the entire text you can add
```
{% if api_resource == "card" %}
  {% assign payeeReferenceMaxLength = 50 %}
{% else %}
  {% assign payeeReferenceMaxLength = 30 %}
{% endif %}
```
and use the `{{ payeeReferenceMaxLength }}` variable to print it out:
` string({{ payeeReferenceMaxLength }})`

**Review by NullableInt — APPROVED** [review]

File: _includes/payee-reference.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 {% assign api_resource = include.api_resource %}
 
+{% if api_resource == "card" %}
+  {% assign payeeReferenceMaxLength = 50 %}
```
Comment (asbjornu) [inline _includes/payee-reference.md:4]:
```suggestion
  {% assign payee_reference_max_length = 50 %}
```

File: _includes/payee-reference.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
...
+{% else %}
+  {% assign payeeReferenceMaxLength = 30 %}
```
Comment (asbjornu) [inline _includes/payee-reference.md:6]:
```suggestion
  {% assign payee_reference_max_length = 30 %}
```

File: _includes/payee-reference.md [hunk 4]
```diff
@@ -18,9 +24,10 @@ specific processing rules depending on specifications in the contract.
 *   Its length and content validation is dependent on whether the
     `transaction.number` or the `payeeReference` is sent to the acquirer.
     1.  If you select *Option A* in the settlement process (Swedbank Pay will
-        handle the settlement), Swedbank Pay will send the `transaction.number` to
-        the acquirer and the `payeeReference` must be in the format of characters
-        `A-Za-z0-9` (including `-`) and `string(30)`.
+        handle the settlement), Swedbank Pay will send the `transaction.number`
+        to the acquirer and the `payeeReference` must be in the format of
+        characters `A-Za-z0-9` (including `-`) and 
+        `string({{ payeeReferenceMaxLength }})`.
```
Comment (asbjornu) [inline _includes/payee-reference.md:30]:
```suggestion
        `string({{ payee_reference_max_length }})`.
```

**Review by asbjornu — CHANGES_REQUESTED** [review]
Variables used in Liquid code should be `snake_cased`. Otherwise this is excellent!

**Review by asbjornu — APPROVED** [review]
👏

**Pull request merged**